File: problemas/1100/1115.py
while True:
    (x, y) = map(int, input().split(' '))

    if x == 0 or y == 0:
        break

    if x > 0 and y > 0:
        quadrante = 'primeiro'
    elif x < 0 and y > 0:
        quadrante = 'segundo'
    elif x < 0 and y < 0:
        quadrante = 'terceiro'
    else:
        quadrante = 'quarto'

    print(quadrante)

File: problemas/1100/1115.js
const entrada = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');

while (true) {
  const [x, y] = entrada.shift().split(' ').map(x => Number(x));

  if (x === 0 || y === 0) {
    break;
  }

  quadrante = x > 0 && y > 0 ?
    'primeiro'
    : x < 0 && y > 0 ?
      'segundo'
      : x < 0 && y < 0 ?
        'terceiro'
        : 'quarto';

  console.log(quadrante);
}